homeassistant/components/inels/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "inels",
+  "name": "iNELS",
+  "codeowners": ["@Nejezchleb", "@jhoralek", "@zed4805"],
+  "config_flow": true,
+  "dependencies": ["mqtt"],
+  "documentation": "https://github.com/zed4805/inels-hacs-new",
```

Comment:
This should point to homeassistants own documentation, please add a documentation PR in the homeassistant docs repo and also brand icons if required, you'll need to link them here in your PR

Reply:
Sorry didn't see you've done this, please update the url